pynamodb/tests/test_model.py

```diff
@@ -580,9 +580,11 @@ def fake_wait(*obj, **kwargs):
                 'ProvisionedThroughput': {
                     'ReadCapacityUnits': 25, 'WriteCapacityUnits': 25
                 },
-                'TableName': 'UserModel'
+                'TableName': 'UserModel',
+                'BillingMode': 'PROVISIONED'
             }
             actual = req.call_args_list[1][0][1]
+            print('actual: ', actual)
```

Comment:
forgotten print?

Reply:
On the requirments changes they have a note:

# we didn't want to bump the actual dependency of the library for consumers as it would effectively
# be a breaking change. As a result, we use the 1.6.0 dependency for development here for the
# purpose of integration tests, even though requirements.txt still has 1.2.0.

so it is not clear they want requirements.txt bumped for everyone. Maybe only users of the new billing system need to bump and there is some explanation in release notes or something. IE you change in the requirements dev but not the main requirements.